MekHQ/unittests/mekhq/campaign/LogEntryTest.java

```diff
@@ -0,0 +1,45 @@
+package mekhq.campaign;
+
```

Comment:
Mixed tabs and spaces, missing a copyright header.

Reply:
Good point, but – now that I look at it – most files have missing or outdated license headers (besides the © year, some are for GPL 2 while the project is GPL 3).

Do you know if there are plans to add license-header management to the build system? (maybe to the new one @neoancient ?)

I'm by no means a license-savvy man, so I'll just add a GPL3 header for https://github.com/licenses/license-templates/blob/master/templates/gpl3-header.txt